app/src/main/java/org/mozilla/fenix/home/intent/DeepLinkIntentProcessor.kt

```diff
@@ -88,6 +89,8 @@ class DeepLinkIntentProcessor(
             "make_default_browser" -> {
                 if (SDK_INT >= Build.VERSION_CODES.N) {
                     val settingsIntent = Intent(Settings.ACTION_MANAGE_DEFAULT_APPS_SETTINGS)
+                    settingsIntent.putExtra(SETTINGS_SELECT_OPTION_KEY, DEFAULT_BROWSER_APP_OPTION)
+                    settingsIntent.putExtra(SETTINGS_SHOW_FRAGMENT_ARGS, bundleOf(SETTINGS_SELECT_OPTION_KEY to DEFAULT_BROWSER_APP_OPTION))
```

Comment:
Can this line be splitted on two rows?
Detekt fails because the 120 characters limit is exceeded.

Reply:
Just fixed it!